.github/workflows/_test-workflows.yaml

```diff
@@ -59,3 +59,18 @@ jobs:
     #   aws_role_name: cicd-iac
     #   image_name: test-docker-build
     #   image_tag: ${{ github.ref_head }}
+
+  test_terraform_deploy_dflook:
+    uses: ./.github/workflows/tf-deploy-dflook.yaml
+    with:
+      aws_account_id: ${{ vars.aws_account_id }}
+      aws_region: ${{ vars.aws_region }}
+      aws_role_name: ${{ vars.aws_role_name }}
+      environment: sandbox
+      github_feedback: true
+      tf_dir: tests/terraform
+      tf_vars: |
+        var1="value1"
```

Comment:
Not sure about the double quotes. But I would keep them and see how they are seen in terraform